src/net/chat.rs

```diff
@@ -2808,6 +2984,287 @@ mod test {
         })
     }
 
+    /// Inner function for testing various kinds of stackerdb/non-stackerdb peer interactions
+    fn inner_convo_handshake_accept_stackerdb(
+        peer_1_services: u16,
+        peer_1_rc_consensus_hash: ConsensusHash,
+        peer_2_services: u16,
+        peer_2_rc_consensus_hash: ConsensusHash,
+    ) {
```

Comment:
Should this go in the `net::stackerdb::tests` module instead?

Reply:
I don't think so.  This is testing the chat behavior for handling handshakes, which is already covered here.